src/scripts/party/Party.ts

```diff
@@ -172,7 +172,7 @@ class Party implements Feature {
         return Math.round(attack * bonus);
     }
 
-    public calculateOnePokemonAttack(pokemon: PartyPokemon, type1: PokemonType = PokemonType.None, type2: PokemonType = PokemonType.None, region: GameConstants.Region = player.region, ignoreRegionMultiplier = false, includeBreeding = false, useBaseAttack = false, overrideWeather: WeatherType, ignoreLevel = false, includeFlute = true): number {
+    public calculateOnePokemonAttack(pokemon: PartyPokemon, type1: PokemonType = PokemonType.None, type2: PokemonType = PokemonType.None, region: GameConstants.Region = player.region, ignoreRegionMultiplier = false, includeBreeding = false, useBaseAttack = false, overrideWeather: WeatherType, ignoreLevel = false, includeFluteZMoves = true): number {
```

Comment:
Could we call `includeFluteZMoves` something like `includeTempBonuses` or something, so we don't end up with a long name, if we add more? And it's also more clear what it should be used for.
(at least that is what i think it does. If i am wrong, ignore me)

Reply:
You are absolutely right. I changed it. I also changed it few lines above as I should have done before...